src/PathMappingRepository.php

```diff
@@ -219,6 +219,10 @@ private function resolveFilesystemPaths($path, $onlyFirst = true)
                 return $this->resolveRelativePaths($filesystemPaths);
             }
 
+            if (null === $filesystemPaths && '/' !== $path) {
```

Comment:
Considering https://github.com/puli/repository/pull/70/files#diff-9eba964c7c0df37591f1971e96d907c4R280 the `/` path shouldn't be considered at all, so I'm adding it to the condition to prevent useless execution